src/object-attribute.scss

```diff
@@ -0,0 +1,47 @@
+@import "./new-settings";
+@import "./mixins";
+
+$fd-object-attribute-text-decoration-none: none;
+$fd-object-attribute-text-decoration-underline: underline;
+
+$block: #{$fd-namespace}-object-attribute;
+
+.#{$block} {
+  @include fd-reset();
+  @include fd-ellipsis();
+
+  color: var(--sapContent_LabelColor);
+  display: block;
+
+  &--link {
+    @include action-cursor();
+
+    text-decoration: $fd-object-attribute-text-decoration-none;
+    color: var(--sapLinkColor);
+
+    &:visited {
+      text-decoration: $fd-object-attribute-text-decoration-none;
+      color: var(--sapLink_Visited_Color);
+    }
+
+    &:hover {
+      @include fd-hover() {
+        text-decoration: $fd-object-attribute-text-decoration-underline;
+        color: var(--sapLink_Hover_Color);
+      }
+    }
+
+    &[disabled],
+    &[aria-disabled="true"] {
+      opacity: var(--sapContent_DisabledOpacity);
+      cursor: not-allowed;
+    }
+
+    &:focus {
+      @include fd-focus();
+      @include fd-fiori-focus($offset: -0.0625rem);
```

Comment:
are you sure this is working in IE11?

Reply:
<img width="629" alt="2020-10-09_21-30-58" src="https://user-images.githubusercontent.com/53487468/95605607-d02d9900-0a76-11eb-8097-0d8b83915d00.png">